tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -1,11 +1,13 @@
-const groupSharePostfix = '\nGroup'
-const userSharePostfix = '\nUser'
-const federationSharePostfix = '\nRemote user'
 const util = require('util')
 const _ = require('lodash')
 const { COLLABORATOR_PERMISSION_ARRAY } = require('../../helpers/sharingHelper')
 const { client } = require('nightwatch-api')
 const collaboratorDialog = client.page.FilesPageElement.SharingDialog.collaboratorsDialog()
+const SHARE_TYPE_STRING = {
+  user: 'user',
+  group: 'group',
+  federation: 'remote'
```

Comment:
why not simply call the key "remote" as well ?

Reply:
In other places in tests, it's called federation so I didn't want to introduce some "new" name.

Reply:
:+1: